Sources/Jobs/JobQueueHandler.swift

```diff
@@ -64,7 +67,8 @@ final class JobQueueHandler<Queue: JobQueueDriver>: Service {
 
     func runJob(_ queuedJob: QueuedJob<Queue.JobID>) async throws {
         var logger = logger
-        logger[metadataKey: "_job_id"] = .stringConvertible(queuedJob.id)
+        let startTime = DispatchTime.now().uptimeNanoseconds
+        logger[metadataKey: "JobId"] = .stringConvertible(queuedJob.id)
```

Comment:
Here you're making each word Uppercased, whereas the other key is snake_cased. I'd prefer sticking to one.

Reply:
> Here you're making each word Uppercased, whereas the other key is snake_cased. I'd prefer sticking to one.

Agreed! Fixed in the latest commit.